frontend/index.html

```diff
@@ -117,5 +100,27 @@
         })
         .forEach((link) => document.head.append(link));
     </script>
+
+    <script
+      type="text/javascript"
+      src="https://preview.vaadin.com/vaadincom/haas-service/haas-loader.js"
```

Comment:
menu links need to be fixed using an URL parameter

Reply:
Done (according to the instructions how to integrate HaaS).